components/nvs_flash/include/nvs_flash.h

```diff
@@ -118,6 +131,21 @@ esp_err_t nvs_flash_erase(void);
  */
 esp_err_t nvs_flash_erase_partition(const char *part_name);
 
+/**
+ * @brief Erase custom partition.
+ *
+ * This function erases all contents of specified custom partition.
```

Comment:
> We try to use imperative

Could you provide an example how this comment should look like?

> And could you also add the note about initialization

I am not sure I can add more because if you will take a look at other comments, you will see:

```
/**
 * @brief Deinitialize NVS storage for the default NVS partition
 *
 * Default NVS partition is the partition with "nvs" label in the partition table.
```

And for other functions you will see the same, all notes just clarify what does "default nvs partition" mean and no more.

Reply:
> Could you provide an example how this comment should look like?
Yes, like this: `Erase all content of specified custom partition.` instead of `Erases all content of specified custom partition.` As I said, though, there are still a lot of functions in IDF which use the older 3rd person style but we're working on that.

Reply:
> I am not sure I can add more because if you will take a look at other comments, you will see:

Sorry, I mean this:

``` 
* @note If the partition is initialized, this function first de-initializes it. Afterwards, the partition has to
*       be initialized again to be used.
```

Reply:
OK, got it and fixed it.